### PR TITLE
Remove debug.keystore info

### PR DESCRIPTION
**Pull request by rpdome**
We already config the test app to use debug.keystore inside the local gradle folder, instead of the default one (Thanks @shahzaibj for pointing out!), so this is no longer needed.

@TylerMSFT  and I agreed that we should drop the text around debug.keystore , both here and in the QuickStart, to prevent confusion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,9 +89,6 @@ To run this sample, you'll need:
 
 ## Steps to Run the app
 
-> [!NOTE]
-> This sample ships with a default `redirect_uri` configured in the `AndroidManifest.xml`. In order for the default `redirect_uri` to work, this project must be built with the `debug.keystore` located in the `gradle/` directory. To configure signing in Android Studio, see [Sign Your App](https://developer.android.com/studio/publish/app-signing).
-
```
Comment (TylerMSFT) [inline README.md]: The part about having a default redirect_uri configured might be helpful. This might be a bit wordy. What do you think of:

As a convenience, this sample ships with a default `redirect_uri` preconfigured in the `AndroidManifest.xml` so that you don't have to first register your own app object. A `redirect_uri` is partly based on your app's signing key. The sample project is configured to sign using the included `debug.keystore` so that the provided `redirect_uri` will work.

Comment (TylerMSFT) [reply README.md]: Oh, and the casing is wrong. It should be "Steps to run the app"

Comment (rpdome) [reply README.md]: Sounds good. will do that.

**Comment by shahzaibj** [comment]
@rpdome It needs to go to the `master` branch
